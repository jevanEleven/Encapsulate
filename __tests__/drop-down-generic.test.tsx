import { render } from '@testing-library/react';
import GenericDropDown from '@/components/ui/drop-down-generic';

describe('GenericDropDown', () => {
  it('renders correctly', () => {
    const { container } = render(
      <GenericDropDown
        stateObject="foo"
        updateState={() => {}}
        options={['Foo', 'Bar', 'Baz']}
        label="Label"
      />
    );

    expect(container).toMatchSnapshot();
  });
});