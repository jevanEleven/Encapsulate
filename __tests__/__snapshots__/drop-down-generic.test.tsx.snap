// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenericDropDown renders correctly 1`] = `
<div>
  <div
    class="flex items-center"
  >
    <span
      class="mr-1 text-sm"
    >
      Label
    </span>
    <select
      class="border border-gray-300 rounded px-1 py-1 text-sm"
    >
      <option
        value="foo"
      >
        Foo
      </option>
      <option
        value="bar"
      >
        Bar
      </option>
      <option
        value="baz"
      >
        Baz
      </option>
    </select>
  </div>
</div>
`;
